### PR TITLE
OCPBUGS-25400: Part 2 - Multi-upgrades should test from nightly to nightly & added missing GCP upgrades in 4.16

### PR DESCRIPTION
**Pull request by jaypoulz**

### CODE REVIEW

**Review by bradmwilliams — APPROVED** [review]

**Review by djslavens — APPROVED** [review]
/lgtm

**Comment by Prashanth684** [comment]
you probabl need to run `make release-controllers` to fix the config

File: core-services/release-controller/_releases/release-ocp-4.16-multi.json [hunk 1]
```diff
@@ -74,6 +76,27 @@
...
+      "prowJob": {
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-stable-4.15-ocp-e2e-upgrade-gcp-ovn-heterogeneous"
+      },
+      "upgrade": true,
+      "upgradeFromRelease": {
+        "candidate": {
+          "stream": "nightly-multi",
+          "version": "4.15"
+        }
+      }
```
Comment (bradmwilliams) [inline core-services/release-controller/_releases/release-ocp-4.16-multi.json:99]: @jaypoulz These reference jobs that have no corresponding definitions.  Either the jobs must be added or these lines removed.

Comment (Prashanth684) [reply core-services/release-controller/_releases/release-ocp-4.16-multi.json:99]: that's correct - reference https://github.com/openshift/release/pull/45417 and copy pasta them for 4.16 I guess

**Review by djslavens — APPROVED** [review]
/lgtm

File: core-services/release-controller/_releases/release-ocp-4.16-multi.json (hunk 1 above)
Comment (jaypoulz) [reply core-services/release-controller/_releases/release-ocp-4.16-multi.json:99]: Yeah I assumed they already existed and started adding them, but never commit all my local changes. :)

Should be passing now, but I'll give the validation checks a chance to run before I confirm anything.

**Comment by jaypoulz** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-e2e-upgrade-gcp-ovn-arm64 periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-stable-4.15-ocp-e2e-upgrade-gcp-ovn-heterogeneous periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-upgrade-gcp-ovn-heterogeneous

**Pull request merged**